File: esbmc.cpp/validador_preco/validador_comprimento_valor.cpp
#define MAX_PRICE_LEN 6

using namespace std;

bool validate_length(int len) {
    __ESBMC_assert(len >= 1 && len <= 5, "Tamanho Incorreto.");
    return true;
}

int main() {
    int len;
    __ESBMC_assume(len >= 1 && len <= 5);

    validate_length(len);

    return 0;
}

File: esbmc.py/Testes02/validador_preco/validador_comprimento_valor.py
MAX_PRICE_LEN: int = 6

def validate_length(length: int) -> bool:
    assert 1 <= length <= 5, "Tamanho Incorreto."
    return True


def main() -> None:
    length: int = int(input()) 

    assert 1 <= length <= 5

    validate_length(length)


main()
